.github/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+<!--
+Thanks for your contribution! Take a moment to answer these questions so that reviewers have the information they need to properly understand your changes:
+
+* What is the current state of things and why does it need to change?
+* What is the solution your changes offer and how does it work?
+
+Are there any issues or other links reviewers should consult to understand this pull request better? For instance:
+
+* Fixes #12345
+* See: #67890
+-->
+
+## Examples
+
+<!--
+Are there any examples of this change being used in another repository?
+
+When considering changes to the MetaMask module template, it's strongly preferred that the change be experimented with in another repository first. This gives reviewers a better sense of how the change works, making it less likely the change will need to be reverted or adjusted later.
+-->
```

Comment:
Great catch I was thinking of updating the PR template after this PR I created an issue here https://github.com/MetaMask/design-tokens/issues/590 but seeing as we are introducing this file now I don't think it would hurt to add it now. Updated here https://github.com/MetaMask/design-tokens/pull/601/commits/5210fff70de4ca3f7a5152f0ea07626746a9603b